pkg/encoding/address/address.go

```diff
@@ -1,21 +1,20 @@
-package crypto
+package address
 
 import (
 	"github.com/CityOfZion/neo-go/pkg/encoding/base58"
 	"github.com/CityOfZion/neo-go/pkg/util"
 )
 
-// AddressFromUint160 returns the "NEO address" from the given
-// Uint160.
-func AddressFromUint160(u util.Uint160) string {
+// EncodeUint160 returns the "NEO address" from the given Uint160.
+func EncodeUint160(u util.Uint160) string {
```

Comment:
May be call it `ToString` or `FromUint160` ?
Looking at previous files it was not completely obvious what `EncodeUint160` it does.

Reply:
`Uint160ToString`?